src/sharedHooks/useTaxonSearch.ts

```diff
@@ -24,9 +24,10 @@ function saveTaxaToRealm( taxa: Taxon[], realm: Realm ) {
   }, "saving remote taxon from useTaxonSearch" );
 }
 
-const useTaxonSearch = ( taxonQuery = "" ) => {
+const useTaxonSearch = ( taxonQueryArg = "" ) => {
   const realm = useRealm( );
   const iconicTaxa = useIconicTaxa( { reload: false } );
+  const taxonQuery = taxonQueryArg.trim();
```

Comment:
comment might be helpful, since I have no idea why we need to trim the query here.

Reply:
Agreed. Added.